.github/workflows/lint.yaml

```diff
@@ -16,19 +16,33 @@ jobs:
     runs-on: "ubuntu-latest"
     steps:
       - uses: "actions/checkout@v3"
-      - uses: "actions/setup-go@v4"
+      - uses: "authzed/actions/setup-go@main"
         with:
-          go-version: "~1.19"
-      - uses: "authzed/actions/gofumpt@main"
-      - uses: "authzed/actions/go-mod-tidy@main"
-      - uses: "authzed/actions/go-generate@main"
-      - uses: "authzed/actions/golangci-lint@main"
+          go-version: "${{ env.GO_VERSION }}"
+      - name: "Lint Go"
```

Comment:
Shouldn't this have a `deps:tidy` step in it to replace the action?

Reply:
ah good catch